pep-0637.rst

```diff
@@ -347,17 +347,12 @@ The successful implementation of this PEP will result in the following behavior:
    - but if no ``**kwargs`` parameter is defined, it is an error.
 
 
-7. Sequence unpacking remains a syntax error inside subscripts::
+7. Sequence unpacking is allowed inside subscripts::
```

Comment:
Note that this requires some specification for what `obj[*()]` should do, which likely has similar semantics to `obj[]`. One option would be an error, another would be to handle it like `obj[kwargs=...]` is handled.

Reply:
Also would probably want to include `obj[*(x,)] == obj[x]` and `obj[*(x,),] == obj[x,]` as corner cases.